ROOTFS/media/rpi/sage-utils/dhcp-pxe/nfs/etc/rc.local

```diff
@@ -3,29 +3,28 @@
 #initializes sd card on rpi
 
 SDDIR="/dev/mmcblk0"
-SDPARTDIR="/dev/mmcblk0p1"
+SDPARTDIR="/dev/mmcblk0p3"
 
 if [ -b "$SDDIR" ]; then
   echo "SD Card found continuing..." | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` > /etc/rc.local.logs
   mkdir -p /media/plugin-data/ | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
   if [ ! -b "$SDPARTDIR" ]; then
         echo "Initializing Partition Table on SD Card" | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
         sed -e 's/\s*\([\+0-9a-zA-Z]*\).*/\1/' << FDISK_CMDS | fdisk /dev/mmcblk0
-                o # clear the in memory partition table
-                g #initialize got partition table
                 n # new partition
-                1 # partition number 1
-                # default - start at beginning of disk
+                p # primary partition
+		3 # partition number 3
+                3637248 # default - start at beginning of free space of disk (after pi os)
                 # default - finish at end of disk
                 w # write the partition table
                 q # and we're done
 FDISK_CMDS
 
-        mkfs -t ext4 /dev/mmcblk0p1 >> /etc/rc.local.logs
+        mkfs -t ext4 /dev/mmcblk0p3 >> /etc/rc.local.logs
   fi
 
   echo "Mounting SD CARD Scratch Space" | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
-  mount /dev/mmcblk0p1 /media/plugin-data | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
+  mount /dev/mmcblk0p3 /media/plugin-data | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
```

Comment:
before mounting can we add a `fsck` check of the partition with the setting to auto-fix any errors (`fsck -a` i think)?

Basically to simulate having a `2` in the `fs_passno` field of an `fstab` (https://www.man7.org/linux/man-pages/man5/fstab.5.html)

https://linux.die.net/man/8/fsck